test_scripts/Smoke/commonSmokeApi.lua

```diff
@@ -0,0 +1,245 @@
+end
+
+function commonSmokeApi.start(pHMIParams, self)
+  self, pHMIParams = commonSmokeApi.getSelfAndParams(pHMIParams, self)
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self)
+  :Do(function()
+      self:initHMI(self)
+      :Do(function()
+          commonFunctions:userPrint(35, "HMI initialized")
+          self:initHMI_onReady(pHMIParams)
+          :Do(function()
+              commonFunctions:userPrint(35, "HMI is ready")
+              self:connectMobile()
+              :Do(function()
+                  commonFunctions:userPrint(35, "Mobile connected")
+                  allowSDL(self)
+                end)
+            end)
+        end)
+    end)
+end
+
+function commonSmokeApi.registerApplicationWithPTU(pAppId, pUpdateFunction, self)
+  self, pAppId, pUpdateFunction = commonSmokeApi.getSelfAndParams(pAppId, pUpdateFunction, self)
+  if not pAppId then pAppId = 1 end
+  self["mobileSession" .. pAppId] = mobile_session.MobileSession(self, self.mobileConnection)
+  self["mobileSession" .. pAppId]:StartService(7)
+  :Do(function()
+      local corId = self["mobileSession" .. pAppId]:SendRPC("RegisterAppInterface",
+        config["application" .. pAppId].registerAppInterfaceParams)
+      EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered",
+        { application = { appName = config["application" .. pAppId].registerAppInterfaceParams.appName } })
+      :Do(function(_, d1)
+          hmiAppIds[config["application" .. pAppId].registerAppInterfaceParams.appID] = d1.params.application.appID
+          EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate",
+            {status = "UPDATE_NEEDED"}, {status = "UPDATING"}, {status = "UP_TO_DATE" })
+          :Times(3)
+          EXPECT_HMICALL("BasicCommunication.PolicyUpdate")
+          :Do(function(_, d2)
+              self.hmiConnection:SendResponse(d2.id, d2.method, "SUCCESS", { })
+              ptu_table = jsonFileToTable(d2.params.file)
+              ptu(self, pAppId, pUpdateFunction)
+            end)
+        end)
+      self["mobileSession" .. pAppId]:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+      :Do(function()
+          self["mobileSession" .. pAppId]:ExpectNotification("OnHMIStatus",
+            {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+          :Times(1)
+          self["mobileSession" .. pAppId]:ExpectNotification("OnPermissionsChange")
+          :Times(AtLeast(1)) -- todo: issue with SDL --> notification is sent twice
+        end)
+    end)
+end
+
+function commonSmokeApi.putFile(params, pAppId, self)
+    if not pAppId then pAppId = 1 end
+    local mobileSession = commonSmokeApi.getMobileSession(pAppId, self);
+    local cid = mobileSession:SendRPC("PutFile", params.requestParams, params.filePath)
+
+    mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+end
+
+
+function commonSmokeApi.postconditions()
+  StopSDL()
+end
+
+return commonSmokeApi
```

Comment:
@AKalinich-Luxoft 
Please add an empty line here

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1697/commits/1f9a96a07f40b1051bd1cd0e5a3b77a979c40def